app/_hub/kong-inc/http-log/how-to/_splunk.md

```diff
@@ -0,0 +1,77 @@
+---
+nav_title: Send Kong Logs to Splunk
+title: Send Kong Logs to Splunk
+---
+
+You can use the HTTP Log plugin to send {{site.base_gateway}} logs to Splunk.
+
+{:.note}
+> **Note**: The following example uses Splunk 9.0.2. If you are using a different version of Splunk,
+check the Splunk documentation for the appropriate method.
+
+## Prerequisites
+
+You have a Splunk authorization token.
```

Comment:
I disagree with this, since shortening it this way would make it inconsistent with the rest of our docs. If you think we should be dropping the "you have xyz" or "your X is set up in Y way" etc, we should bring that up as a style guide topic.